include/NovelRT.Interop/Graphics/NrtBasicFillRect.h

```diff
@@ -11,6 +11,9 @@ extern "C"
 {
 #endif
 
+    // destroy
+    NrtResult Nrt_BasicFillRect_destroy(NrtBasicFillRectHandle rect);
```

Comment:
Conventionally, destroy methods are the last method of the type to be defined. Can we do that here and in the other places you have added these? And can we please make the cpp files mirror these declarations?

Reply:
Should be done now.